lib/modem_jwt/Kconfig

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig MODEM_JWT
+	bool "Modem JWT Library"
+	select NRF_MODEM_LIB
+	help
+		Functionality requires modem firmware version 1.3 or greater.
```

Comment:
my understanding of the proper formatting of help within a Kconfig file is that, just like the other lines above it, it is indented by 1 tab, but also for help text itself, by an additional two spaces -- not another tab

Reply:
it seems like both are allowed.  
i referenced `lte_link_control`, which uses a tab.  
`hw_unique_key` uses 2 spaces.
🤷

Reply:
I got pushback on a PR a while ago for this, and was told in no uncertain terms that it was 1 tab and 2 spaces.  The file I had changed was inconsistent internally, so I ended up changing the whole file to comply.  The PR was accepted after that.

See:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/kconfig/tips.html
- Use a single tab for each indentation
- Indent help text with two extra spaces